assets/components/service-ca/deployment.yaml

```diff
@@ -23,45 +22,45 @@ spec:
         app: service-ca
         service-ca: "true"
     spec:
-      serviceAccount: service-ca
       serviceAccountName: service-ca
       containers:
-      - name: service-ca-controller
-        image: {{ .ReleaseImage.service_ca_operator }}
-        imagePullPolicy: IfNotPresent
-        command: ["service-ca-operator", "controller"]
-        ports:
-        - containerPort: 8443
-        securityContext:
-          runAsNonRoot: true
-        resources:
-          requests:
-            memory: 120Mi
-            cpu: 10m
-        volumeMounts:
-        - mountPath: /var/run/secrets/signing-key
-          name: signing-key
-        - mountPath: /var/run/configmaps/signing-cabundle
-          name: signing-cabundle
+        - name: service-ca-controller
+          image: {{ .ReleaseImage.service_ca_operator }}
+          imagePullPolicy: IfNotPresent
+          command: ["service-ca-operator", "controller"]
+          ports:
+            - containerPort: 8443
+          securityContext:
+            runAsNonRoot: true
+          resources:
+            requests:
+              memory: 120Mi
+              cpu: 10m
+          volumeMounts:
+            - mountPath: /var/run/secrets/signing-key
+              name: signing-key
+            - mountPath: /var/run/configmaps/signing-cabundle
+              name: signing-cabundle
       volumes:
-      - name: signing-key
-        secret:
-          secretName: {{.TLSSecret}}
-      - name: signing-cabundle
-        configMap:
-          name: {{.CAConfigMap}}
-      # nodeSelector:
-      #   node-role.kubernetes.io/master: ""
+        - name: signing-key
+          secret:
+            secretName: {{.TLSSecret}}
+        - name: signing-cabundle
+          configMap:
+            name: {{.CAConfigMap}}
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
       priorityClassName: "system-cluster-critical"
       tolerations:
-      - key: node-role.kubernetes.io/master
-        operator: Exists
-        effect: "NoSchedule"
-      - key: "node.kubernetes.io/unreachable"
-        operator: "Exists"
-        effect: "NoExecute"
-        tolerationSeconds: 120
-      - key: "node.kubernetes.io/not-ready"
-        operator: "Exists"
-        effect: "NoExecute"
-        tolerationSeconds: 120
+        - key: node-role.kubernetes.io/master
```

Comment:
again - these whitespaces don't appear in the orig

Reply:
The whitespace change is the result from `yq`'s reformatting.